trade-finance-manager-ui/server/controllers/case/underwriting/lead-underwriter/helpers.js

```diff
@@ -0,0 +1,14 @@
+import CONSTANTS from '../../../../constants';
+import { userIsInTeam } from '../../../../helpers/user';
+
+const canUserEditLeadUnderwriter = (user) => {
+  const isManager = userIsInTeam(user, [CONSTANTS.TEAMS.UNDERWRITER_MANAGERS]);
+
+  if (isManager) {
```

Comment:
why not use `return userIsInTeam(user, [CONSTANTS.TEAMS.UNDERWRITER_MANAGERS])` ?

Reply:
Updated, not sure why I didn't simplify it